tests/sceneManager_test.cpp

```diff
@@ -3,14 +3,64 @@
 #include "obj/objects.h"
 #include "gtest/gtest.h" // https://google.github.io/styleguide/cppguide.html#Namespaces
 
+// TODO: change underscore to the end
+
 using core::Ray;
 using core::RayHitData;
 using core::Vec3;
 using generators::PointSource;
 using objects::Object;
 using objects::TriangleObj;
 
-const double kSkipFreq = 1000;
 generators::FakeRandomGen fakeRand;
 
-// TODO: Change all doubles to float
+class SceneManagerTest : public ::testing::Test
+{
+protected:
+    SceneManager manager;
```

Comment:
Please take a look again at what is the proper order of members in the class, and fix this accordingly.

Reply:
done